x-pack/platform/plugins/shared/observability_ai_assistant/server/service/inference_endpoint.ts

```diff
@@ -123,7 +139,7 @@ export async function getElserModelStatus({
     });
 
   if (!modelStats) {
-    return { ready: false, enabled, errorMessage };
```

Comment:
I think Saander added `enabled`. I don't think we ever used it though. Does the search team use it for anything?

Reply:
enabled is now returned higher up, but @sphilipse maybe you (or your team) can verify?